custom/icds_reports/utils/data_pull/governance_dashboard.sql

```diff
@@ -0,0 +1,177 @@
+                                             ->  Parallel Seq Scan on "agg_awc_2019-11-01_5" agg_awc_5  (cost=0.00..112074.08 rows=141670 width=173)
+                                                   Filter: ((month = '2019-11-01'::date) AND (num_launched_awcs = 1))
+                                             ->  Parallel Seq Scan on "agg_awc_2019-11-01_4" agg_awc_4  (cost=0.00..1716.89 rows=185 width=144)
+                                                   Filter: ((month = '2019-11-01'::date) AND (num_launched_awcs = 1))
+                                             ->  Parallel Seq Scan on "agg_awc_2019-11-01_3" agg_awc_3  (cost=0.00..253.78 rows=7 width=115)
+                                                   Filter: ((month = '2019-11-01'::date) AND (num_launched_awcs = 1))
+                                             ->  Parallel Seq Scan on "agg_awc_2019-11-01_2" agg_awc_2  (cost=0.00..23.42 rows=1 width=86)
+                                                   Filter: ((month = '2019-11-01'::date) AND (num_launched_awcs = 1))
+                                             ->  Parallel Seq Scan on "agg_awc_2019-11-01_1" agg_awc_1  (cost=0.00..2.30 rows=1 width=168)
+                                                   Filter: ((month = '2019-11-01'::date) AND (num_launched_awcs = 1))
+                                             ->  Parallel Seq Scan on agg_awc  (cost=0.00..0.00 rows=1 width=168)
+                                                   Filter: ((month = '2019-11-01'::date) AND (num_launched_awcs = 1))
+                                       ->  Parallel Hash  (cost=69076.10..69076.10 rows=185910 width=240)
+                                             ->  Parallel Seq Scan on awc_location_local awc_location  (cost=0.00..69076.10 rows=185910 width=240)
+                           ->  Seq Scan on icds_months_local months  (cost=0.00..25.00 rows=6 width=4)
+                                 Filter: (start_date = '2019-11-01'::date)
+                     ->  Append  (cost=0.00..1772038.06 rows=8 width=50)
+                           ->  Seq Scan on agg_child_health  (cost=0.00..0.00 rows=1 width=80)
+                                 Filter: ((aggregation_level = 5) AND (month = '2019-11-01'::date) AND (awc_location.doc_id = awc_id) AND (awc_location.aggregation_level = aggregation_level))
+                           ->  Index Scan using staging_agg_child_health_aggregation_level_gender_idx1 on "agg_child_health_2019-11-01"  (cost=0.56..1772038.02 rows=7 width=50)
+                                 Index Cond: ((awc_location.aggregation_level = aggregation_level) AND (aggregation_level = 5))
+                                 Filter: ((month = '2019-11-01'::date) AND (awc_location.doc_id = awc_id))
+               ->  Result  (cost=0.00..0.00 rows=0 width=0)
+                     One-Time Filter: false
+(38 rows)
+
+*/
+
+
+
+-- to get pw and lw data for nov and Oct in one sheet
```

Comment:
same question about combining in python here, although this is obviously a less costly query

Reply:
Cool. I think that makes sense.

Reply:
@Cal done